components/scream/src/physics/shoc/shoc_functions_f90.cpp

```diff
@@ -46,6 +46,10 @@ void eddy_diffusivities_c(Int nlev, Int shcol, Real *obklen, Real *pblh,
 
 void calc_shoc_vertflux_c(Int shcol, Int nlev, Int nlevi, Real *tkh_zi,
 			  Real *dz_zi, Real *invar, Real *vertflux);
+
+void shoc_diag_second_moments_srf_c(Int shcol, Real* wthl, Real* uw, Real* vw, 
+                          Real* ustar2, Real* wstar);
```

Comment:
This second line should line up with the first.

Reply:
yes, done!